src/app/tests/suites/certification/Test_TC_WNCV_4_1.yaml

```diff
@@ -0,0 +1,220 @@
+                value: 2500
+
+    - label: "Wait 1000ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 1000
+
+    - label:
+          "If (PA_LF & LF) TH reads TargetPositionLiftPercent100ths attribute
+          from DUT"
+      disabled: true
+      command: "readAttribute"
+      attribute: "TargetPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          value: 2500
+
+    - label:
+          "If (PA_LF & LF) TH reads TargetPositionLiftPercent100ths attribute
+          from DUT"
+      command: "readAttribute"
+      attribute: "TargetPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          constraints:
+              type: percent100ths
+              minValue: 1
+              maxValue: 10000
+
+    - label: "TH set a timeout of 5-10 minutes for failure"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 5000
+    #Need Bitwise operator support #14491
+    - label: "TH reads OperationalStatus attribute from DUT"
+      command: "readAttribute"
+      attribute: "OperationalStatus"
+      response:
+          constraints:
+              type: map8
+
+    - label:
+          "If (PA_LF & LF) TH reads CurrentPositionLiftPercent100ths attribute
+          from DUT"
+      command: "readAttribute"
+      attribute: "CurrentPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          constraints:
+              type: percent100ths
+              minValue: 1
+              maxValue: 10000
+
+    - label:
+          "If (PA_LF & LF) TH reads CurrentPositionLiftPercentage attribute from
+          DUT"
+      command: "readAttribute"
+      attribute: "CurrentPositionLiftPercentage"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          constraints:
+              type: percent
+              minValue: 0
+              maxValue: 100
+
+    #Disabled due to issue #14230
+    - label: "TH sends GoToLiftPercentage command to DUT"
+      disabled: true
+      command: "GoToLiftPercentage"
+      arguments:
+          values:
+              - name: "LiftPercentageValue"
+                value: 75
+              - name: "LiftPercent100thsValue"
+                value: 7500
+
+    - label: "Wait 1000ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 1000
+
+    - label:
+          "If (PA_LF & LF) TH reads TargetPositionLiftPercent100ths attribute
+          from DUT"
+      disabled: true
+      command: "readAttribute"
+      attribute: "TargetPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          value: 7500
+
+    - label: "TH waits for 5-15 seconds movement(s) on the DUT"
```

Comment:
```suggestion
    - label: "TH waits for 5-10 minutes movement(s) on the DUT"
```

Reply:
should be a parameter for the tests set by manufacturer 
it is not clear in the test plan i know